frontend/ui/widget/menu.lua

```diff
@@ -648,7 +648,7 @@ function Menu:updateItems(select_number)
                 state_size = self.state_size or {},
                 text = self.item_table[i].text,
                 mandatory = self.item_table[i].mandatory,
-                bold = self.item_table.current == i,
+                bold = self.item_table.current == i or self.item_table[i].bold ~= nil,
```

Comment:
`bold` will be evaluated to `true` when `item_table[i]` is `false`. Better change it to `== true`